src/autoCollection/metrics/httpMetricsInstrumentation.ts

```diff
@@ -2,24 +2,27 @@
 // Licensed under the MIT license.
 import type * as http from 'http';
 import type * as https from 'https';
+export type Http = typeof http;
 import * as semver from 'semver';
 import * as url from 'url';
 import {
   InstrumentationBase,
-  InstrumentationConfig,
   InstrumentationNodeModuleDefinition,
   isWrapped,
   safeExecuteInTheMiddle
 } from '@opentelemetry/instrumentation';
 import { getRequestInfo } from '@opentelemetry/instrumentation-http';
+import { Histogram, MeterProvider, ValueType } from '@opentelemetry/api-metrics';
 
 import { APPLICATION_INSIGHTS_SDK_VERSION } from "../../declarations/constants";
-import { IHttpMetric, IMetricDependencyDimensions, IMetricRequestDimensions } from './types';
+import { HttpMetricsInstrumentationConfig, IHttpStandardMetric, StandardMetric } from './types';
+import { Logger } from '../../library/logging';
+import { SpanKind } from '@opentelemetry/api';
+import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
 
 
-export class HttpMetricsInstrumentation extends InstrumentationBase {
+export class HttpMetricsInstrumentation extends InstrumentationBase<Http> {
 
-  private static _instance: HttpMetricsInstrumentation;
   private _nodeVersion: string;
   public totalRequestCount: number = 0;
```

Comment:
Why do we need to keep track of these totals? Are you planning to replace [performance counters](https://github.com/microsoft/ApplicationInsights-node.js/blob/develop/AutoCollection/Performance.ts) with this instrumentation?

Reply:
Perfomance counters are currently calculated using these, is still not very clear how perf counters would be handled in OpenTelemetry so having this functionality in the meantime